### PR TITLE
Update Macos Instructions to use new actions

### PR DESCRIPTION
**Pull request by Spidy123222**
Hopefully, this works for people.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,28 +94,23 @@ where you replace UDID and PackageID with their values you found before.
...
 
-2. Move the two idevice files to 
+	[Download](https://github.com/libimobiledevice/libimobiledevice/actions)
 
-`` /usr/local/bin ``
+2. In terminal enter 
 
-3. Move the dylib to 
-`` /usr/local/lib ``
+`` chmod +x ~/Downloads/libimobiledevice-latest_macOS/libimobiledevice.tar ``
```
Comment (jkcoxson) [inline README.md:103]: I was saying that the bins have to be chmod-ed, not the tar

File: README.md [hunk 2]
```diff
@@ -94,28 +94,23 @@ where you replace UDID and PackageID with their values you found before.
...
+3. Make sure your device is plugged in and that it is Trusted. Then go in Finder and click your device and Enable WiFi sync.
 
-`` chmod +x /usr/local/bin/idev* ``
-
-5. Make sure your device is plugged in and that it is Trusted. Then go in Finder and click your device and Enable WiFi sync.
-
-6. Type in and enter 
+4. Type in and enter 
 
 `` idevice_id ``
```
Comment (jkcoxson) [inline README.md:114]: The bins either need to be moved to usr lib or a cd and ./ needs to be added

**Review by jkcoxson — CHANGES_REQUESTED** [review]
I've commented some changes

File: README.md (hunk 2 above)
Comment (Spidy123222) [reply README.md:114]: So I could just make it Chmod the bins file in the tar file and work out fine, Does that sound correct?

Comment (jkcoxson) [reply README.md:114]: Like cd to the directory with the binaries (like idevice_id etc) and chmod +x *

File: README.md (hunk 1 above)
Comment (Spidy123222) [reply README.md:103]: > I was saying that the bins have to be chmod-ed, not the tar

will resolve

File: README.md (hunk 2 above)
Comment (Spidy123222) [reply README.md:114]: So something in the lines of.

cd ~/Downloads/libimobiledevice-latest_macOS/usr/local/bin/
chmod +x /idev*

Comment (jkcoxson) [reply README.md:114]: If that's how the folder is structured, then yes

**Pull request merged**